lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -433,6 +455,71 @@ describe('BasicLtiLaunchApp', () => {
       await waitForElement(wrapper, 'FakeDialog[title="Authorize Hypothesis"]');
       await contentHidden(wrapper);
     });
+
+    context('when auth fails multiple times in a row', () => {
+      // Helper to ensure the dialog is initially present but also remains so
+      // for a short duration longer without disappearing.
+      async function dialogShouldRemain(wrapper) {
+        // Add an artificial delay to ensure that all the api calls resolve or
+        // reject before proceeding.
+        await true;
+        await true;
+        // Error prompt should remain.
+        assert.isTrue(wrapper.find('FakeDialog').exists());
+        // Error prompt should also not go away after a short while (e.g.flicker)
+        try {
+          await waitFor(() => {
+            wrapper.update();
+            const el = wrapper.find('FakeDialog');
+            if (el.length === 0) {
+              throw new Error();
```

Comment:
I would recommend to always set the `message` parameter when creating an `Error` to make debugging easier.

Edit: Oh I see, you're using this to test what error was reported in the "catch" clause.